library/vtproto/v0.3.0/buf.plugin.yaml

```diff
@@ -0,0 +1,13 @@
+version: v1
+name: buf.build/library/vtproto
+plugin_version: v0.3.0
+output_languages: [go]
+source_url: https://github.com/planetscale/vtprotobuf
+description: Generates optimized marshall & unmarshal code.
+default_opts:
+  - paths=source_relative
```

Comment:
I think we may want to set additional options (i.e. disable the experimental gRPC feature by default): https://github.com/planetscale/vtprotobuf/issues/25#issuecomment-1011055026